ding/data/buffer/deque_buffer_wrapper.py

```diff
@@ -61,7 +61,7 @@ def __init__(
             self.last_sample_index = None
             self.last_sample_meta = None
 
-    def sample(self, size: int, train_iter: int):
+    def sample(self, size: int, train_iter: int = -1):
```

Comment:
Why here use -1 as the default value, instead of 0?

Reply:
fixed it